backend/services/logs/src/bullmq/afterProcess.ts

```
@@ -0,0 +1,261 @@
+        handleAfterMutation(props);
+        continue;
+      }
+
+      if (source === 'auth') {
+        handleAfterAuth(props);
+        continue;
+      }
+
+      if (source === 'webhook') {
+        handleAfterAPIRequest(props);
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)